libs/shared-lib/src/lib/Tabs/CommonTabs.tsx

```
@@ -0,0 +1,77 @@
+            <Tab
+              key={index}
+              icon={tab.icon ?? undefined}
+              label={tab.label}
+              {...a11yProps(index)}
+            />
+          ))}
+        </Tabs>
+      </Box>
+
+      {tabs.map((tab, index) => (
+        <CustomTabPanel value={value} index={index}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)